Modules/faulthandler.c

```diff
@@ -1372,7 +1379,18 @@ _PyFaulthandler_Init(int enable)
        SIGSTKSZ bytes. Calling the previous signal handler in faulthandler
        signal handler uses more than SIGSTKSZ bytes of stack memory on some
        platforms. */
+#ifdef AT_MINSIGSTKSZ
+    {
+        unsigned long at_minstack_size = getauxval(AT_MINSIGSTKSZ);
+        if (at_minstack_size > SIGSTKSZ) {
+            stack.ss_size = SIGSTKSZ + at_minstack_size;
+        } else {
+            stack.ss_size = SIGSTKSZ * 2;
+        }
+    }
+#else
     stack.ss_size = SIGSTKSZ * 2;
+#endif
```

Comment:
Please add a comment to explain why you use AT_MINSIGSTKSZ. I suggest to change the code like that:

```suggestion
    stack.ss_size = SIGSTKSZ * 2;

#ifdef AT_MINSIGSTKSZ
    // bpo-46968: (...)
    unsigned long at_minstack_size = getauxval(AT_MINSIGSTKSZ);
    if (at_minstack_size != 0) {
        stack.ss_size = SIGSTKSZ + at_minstack_size;
    }
#endif
```

Reply:
Done. Thank you for the review.